src/diffusers/models/unet_2d_blocks.py

```diff
@@ -1668,8 +1674,10 @@ def custom_forward(*inputs):
                     create_custom_forward(attn, return_dict=False),
                     hidden_states,
                     encoder_hidden_states,
+                    None,  # emb
```

Comment:
```suggestion
                    temb,
```

I think this was never tested :sweat_smile: would be great if we could add it here